filter-parser/src/value.rs

```diff
@@ -7,8 +7,54 @@ use nom::sequence::{delimited, terminated};
 use crate::error::NomErrorExt;
 use crate::{parse_geo_point, parse_geo_radius, Error, ErrorKind, IResult, Span, Token};
 
+/// This function goes through all chacaters in the [Span], if it finds any escaped character (`\`).
+/// It generate a new string with all `\` removed from the [Span].
+fn unescape(buf: Span, char_to_escape: char) -> String {
+    let to_escape = format!("\\{}", char_to_escape);
+    buf.replace(&to_escape, &char_to_escape.to_string())
```

Comment:
Yeah, if someone writes `\"` he'll end up with `"`.
But if he wrote `\\"` since we apply the replace only once, he'll end up with `\"`, which should be okay, I guess.
And everything else stays untouched.

Reply:
Hum, I understand now!